herddb-core/src/test/java/herddb/index/blink/BlinkBench.java

```diff
@@ -28,7 +28,7 @@
 import java.io.PrintStream;
 import java.io.PrintWriter;
 import java.io.StringWriter;
-import java.nio.charset.StandardCharsets;
+import java.io.UnsupportedEncodingException;
```

Comment:
Revert changes introduced by code inspector. The method is added in Java 10 and we have to support 1.8 though, although we're in test scope.